src/tests/ftest/daos_test/suite.py

```diff
@@ -408,3 +408,17 @@ def test_daos_dedup(self):
         :avocado: tags=daos_test,daos_core_test,test_daos_dedup
         """
         self.run_subtest()
+    def test_daos_upgrade(self):
```

Comment:
There should be a newline before this definition

Reply:
sure